plugins/modules/dcnm_policy.py

```diff
@@ -927,7 +930,7 @@ def dcnm_policy_delete_policy(self, policy, mark_del):
             command = "PUT"
         else:
             #path = "/rest/control/policies/" + policy
-            path = self.paths["POLICY_WITH_POLICY"].format(policy)
+            path = self.paths["POLICY_WITH_POLICY_ID"].format(policy)
```

Comment:
What is the reason for changing from `POLICY_WITH_POLICY` to `POLICY_WITH_POLICY_ID`?

Reply:
It is actually getting Policy information using POLICY-ID. So changed it to reflect it correctly